custom_components/octopus_energy/api_client/__init__.py

```diff
@@ -390,22 +390,25 @@
   octoHeatPumpUpdateFlowTemperatureConfiguration(
     euid: "{euid}"
     flowTemperatureInput: {{
-      useWeatherCompensation: {use_weather_comp}, 
-      flowTemperature: {
+      useWeatherCompensation: {weather_comp_enabled}, 
+      flowTemperature: {{
```

Comment:
question: I assume it doesn't error when both are supplied regardless of if useWeatherCompensation is true or false? I only ask as the API looks like it expects `flowTemperature` to be null if `useWeatherCompensation` is false and `weatherCompensationValues` to be null if `useWeatherCompensation` is true.